### PR TITLE
Added more songs in Openings & OST

### PR DESCRIPTION
**Pull request by saideepd**
Made some additions as per #195 
#### This PR is for [Re:Zero Kara Hajimeru Isekai Seikatsu, Bleach, Detective Conan, One Piece, Inazuma Eleven, Death Note].

- [x] This issue is for enhancement/bug fixing.
- [x] This issue is an addition to JSON database.
	- [x] Pull Request Title is of form `Added [Re:Zero Kara Hajimeru Isekai Seikatsu, Bleach, Detective Conan, One Piece, Inazuma Eleven, Death Note]`.
- [ ] This addition is already on ongaku.
	- [ ] This PR is an improvement to current addition.
- [x] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/anshumanv/ongaku/blob/master/CONTRIBUTING.md).
- [ ] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

**Comment by saideepd** [comment]
Added more Opening & OST, fixed some bugs.

**Comment by saideepd** [comment]
Sorry for closing the pull request by mistake. My branch has passed all the tests. Please merge it with master branch.

**Comment by anshumanv** [comment]
Thanks, @saideepd for the PR.
I'll merge once reviewed. 😄

File: js/data.json [hunk 1]
```diff
@@ -1280,6 +1292,30 @@ osts = [
 		"link" : "https://s1.vocaroo.com/media/download_temp/Vocaroo_s1oMIbR1BFgl.mp3",
 		"name" : "Kimi no na wa OST - Zen zen zense Radwimps",
 		"img" : "https://i.imgur.com/QIyA637.png"
+	},
+	
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/3l41690wfwq3vvo/real%20bleach%20opening%201%20orange%20range%20asterisk.m4a?dl=0",
+		"name" : "Bleach OST 1 - Asterisk - Orange Range",
+		"img" : "https://i.ytimg.com/vi/LGAYnGcOd2o/maxresdefault.jpg"
```
Comment (anshumanv) [inline js/data.json]: Use some better image like an anime scene or something, also this is OP1 so please place this in openings section.

File: js/data.json [hunk 2]
```diff
@@ -520,6 +520,18 @@ openings = [
...
+	
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/g95vnwebcmb93nv/Re-Zero%20Kara%20Hajimeru%20Isekai%20Seikatsu%20Opening%20FULL.mp3?dl=0",
+		"name" : "Re:Zero Kara Hajimeru Isekai Seikatsu OST - Opening 1",
+		"img" : "https://image.tmdb.org/t/p/original/n6YYb2jMN0UE1YNJcPSHnwX7vMq.jpg"
+	},
+	
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/27m8fylvfa42gsa/Inazuma%20Eleven%20Opening%20Theme%20Song.m4a?dl=0 ",
+		"name" : "Inazuma Eleven OST - Opening Theme Song",
```
Comment (anshumanv) [inline js/data.json]: Isn't this OP ?

File: js/data.json [hunk 3]
```diff
@@ -1280,6 +1292,30 @@ osts = [
...
+	
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/3l41690wfwq3vvo/real%20bleach%20opening%201%20orange%20range%20asterisk.m4a?dl=0",
+		"name" : "Bleach OST 1 - Asterisk - Orange Range",
+		"img" : "https://i.ytimg.com/vi/LGAYnGcOd2o/maxresdefault.jpg"
+	},
+	
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/9sqnxnojdtrcjnm/Detective%20Conan%20Soundtrack%201.m4a?dl=0",
+		"name" : "Detective Conan OST 1",
```
Comment (anshumanv) [inline js/data.json]: Does this OST have a name? If so please add it

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please check comments

**Comment by saideepd** [comment]
Sure. Thanks @anshumanv for the reply. Hope that my PR will get merged 😄

**Comment by anshumanv** [comment]
I am seeing that you're using dropbox links, are you sure the links will stay in place?

**Comment by saideepd** [comment]
Yes. I have hosted them on my personal Dropbox account. And I don't use Dropbox a lot, so I won't be deleting them.

**Comment by anshumanv** [comment]
Alright then, I'll merge after you make requested changes. 😄

**Comment by saideepd** [comment]
I have made changes as requested. Please review it now.

**Review by anshumanv — APPROVED** [review]

**Comment by saideepd** [comment]
@anshumanv Thanks for merging the PR bro 👍

**Comment by anshumanv** [comment]
Thanks for the PR @saideepd, you can continue to add more if you wish. 😄

**Comment by saideepd** [comment]
Sure 👍 @anshumanv

**Pull request merged**